--- org/script.artistslideshow/default.py	2012-02-24 16:11:05.000000000 -0500
+++ new/script.artistslideshow/default.py	2012-02-24 16:14:12.000000000 -0500
@@ -234,9 +234,13 @@
 
             path = getCacheThumbName(url, self.CacheDir)
             if not xbmcvfs.exists(path):
-                download(url, path)
-                log('downloaded %s to %s' % (url, path) )
-                self.ImageDownloaded=True
+                try:
+                    download(url, path)
+                except:
+                    log ('site unreachable')
+                else:
+                    log('downloaded %s to %s' % (url, path) )
+                    self.ImageDownloaded=True
             if self.ImageDownloaded:
                 if not self.DownloadedFirstImage:
                     log('downloaded first image')
